docs/uml-class-diagrams/admin/test/SatelliteTests/SatelliteTests.puml

```diff
@@ -2,7 +2,14 @@
 
 package edu.oakland.test.admin {
     class SatelliteTests {
-        
+       + void satNameInIsSatNameOut()
+       + void strengthInIsStrengthOut()
+       + void locInIsLocOut()
```

Comment:
I don't think we need this one, as the location that we store in is the initial location, and the location out is the one being generated.